packages/math-input/src/components/provided-keypad.tsx

```diff
@@ -14,17 +14,17 @@ import {createStore} from "../store/index";
 
 import KeypadContainer from "./keypad-container";
 
+import type {Cursor, KeypadConfiguration, KeyHandler} from "../types";
 import type {StyleType} from "@khanacademy/wonder-blocks-core";
 
 type Props = {
     onElementMounted?: (arg1: any) => void;
-    onDismiss?: () => unknown;
+    onDismiss?: () => void;
     style?: StyleType;
 };
 
 class ProvidedKeypad extends React.Component<Props> {
-    // @ts-expect-error [FEI-5003] - TS2564 - Property 'mounted' has no initializer and is not definitely assigned in the constructor.
-    mounted: boolean;
+    mounted?: boolean;
```

Comment:
**nit**: I am guessing our code doesn't care between `undefined` and `false` so we can avoid initializing it, but I feel its slightly more accurate/clear to initialize it here and then make it non-optional. 

```suggestion
    mounted: boolean = false;
```